site/gulpfile.js

```diff
@@ -1,18 +1,20 @@
-const del           = require('del');
-const gulp          = require('gulp');
-const cachebust     = require('gulp-rev');
-const cacheClean    = require('gulp-rev-delete-original');
-const babel         = require('gulp-babel');
-const uglify        = require('gulp-uglify-es').default;
-const pug           = require('gulp-pug');
-const htmlmin       = require('gulp-htmlmin');
-const stylus        = require('gulp-stylus');
-const webserver     = require('gulp-webserver');
-const concat        = require('gulp-concat');
-const download      = require('gulp-download-stream');
-const responsive    = require('gulp-responsive');
-
-const {swagList, swagImages} = require('./get-data');
+const del               = require('del');
```

Comment:
I think we need to just use one space to separate equal signs

```js
const package = require('package');
const superLongPackageName = require('superLongPackageName');
```

Otherwise, every time we modify our dependencies, the diffs will be unnecessarily bloated and I don't want to click extra buttons 😆

Reply:
I'm super ok with what you saying but as we discussed said in here (https://github.com/swapagarwal/swag-for-dev/pull/76#issuecomment-433082739) we should settle for coding conventions in another PR. I tried to stick with the current one in this PR so.

Reply:
Of course I was the one that said something 😅 I meant this more as an observation rather than something that should be changed 👍